flask_app/models.py

```diff
@@ -80,9 +81,31 @@ def test_metadata(self):
                 combined_json_object[key] = value
         return combined_json_object
 
+    @computed_field
```

Comment:
This should be removed. The specific test route should fetch all test errors, while the list of tests should only contain the number of errors, which is scalar...

The current implementation performs N^2 queries, and for large numbers of tests this could be painful. In any case, there's no need for this because in the "all tests" view the exception information is not really useful...